tests/src/test/scala/org/json4s/ext/JodaTimeSerializerSpec.scala

```diff
@@ -53,13 +53,13 @@ abstract class JodaTimeSerializerSpec(mod: String) extends Specification {
         }
       } ++ JodaTimeSerializers.all
 
-      val x = Dates(new DateTime(2011, 1, 16, 10, 32, 0, 0), new DateMidnight(2011, 1, 16))
+      val x = Dates(new DateTime(2011, 1, 16, 10, 32, 0, 0), new DateMidnight(2011, 1, 16), new LocalDateTime(2015, 3, 12, 11, 55, 0, 0))
       val ser = s.write(x)
-      ser must_== """{"dt":"2011-01-16 10:32:00Z","dm":"2011-01-16 00:00:00Z"}"""
+      ser must_== """{"dt":"2011-01-16 10:32:00Z","dm":"2011-01-16 00:00:00Z","ldt":"2015-03-12 11:55:00Z"}"""
```

Comment:
Agreed, LocalDateTime doesn't have timezone. So looks like Z has to be removed from the string representation.